submissions/olabodeAmusan/Database/init.sql

```diff
@@ -0,0 +1,58 @@
+SHOW DATABASES;
+CREATE DATABASE foodbag;
+/*Table for four different food categories*/
+USE foodbag;
+CREATE TABLE food (
+id INT NOT NULL AUTO_INCREMENT,
+type VARCHAR(250),
+PRIMARY KEY(id)
+);
+
+INSERT INTO food (type)
+VALUES('fast food'),
+('vegetables & fruits'),
+('drinks & cocktail'),
+('resturant');
+
+/*Table showing the relationship between the vendor itself and the food they sell*/ 
+CREATE TABLE vendors_food (
+id INT NOT NULL AUTO_INCREMENT,
+admin_id INT NOT NULL,
+food_id INT NOT NULL,
+name VARCHAR(250),
+amount DECIMAL(10,2) NOT NULL,
+PRIMARY KEY(id)
+);
+
+/*Table for Admins*/
+CREATE TABLE admins (
+id INT NOT NULL AUTO_INCREMENT,
+name VARCHAR(250),
+type INT NOT NULL,
```

Comment:
The admins' table will contain both the system admins of the food bag platform and the vendors. They both will be differentiated by type. The system admins can have a type of 0 and vendors a type of 1 when they are saved for differentiation. 

Boolean values are nothing but 0 and 1 i.e. either true or false. if the condition satisfies, it is true else it is false. So change line 3 to:  type BOOLEAN default TRUE.